porch/internal/api/porchinternal/v1alpha1/config.porch.kpt.dev_packagerevs.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
+  creationTimestamp: null
+  name: packagerevs.config.porch.kpt.dev
```

Comment:
Nit: It's only a temporary name, I guess, but we should generally avoid abbreviations IMO

Reply:
Yeah, I would rather have called it `packagerevisions`, but that leads to issues since it is already in use. I was thinking that if/when we can remove the `packagerevisions` resource from the aggregated apiserver, we could introduce an packagerevisions alias for this one.